packages/react-select/.storybook/tsconfig.json

```
@@ -0,0 +1,10 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "",
+    "allowJs": true,
+    "checkJs": true,
+    "types": ["static-assets", "environment", "inline-style-expand-shorthand", "storybook__addons"]
```

```suggestion
    "types": ["static-assets", "environment", "storybook__addons"]
```

We don't have typings for `inline-style-expand-shorthand`, surprised if it passing CI at all :)